packages/shared/components/popups/SendFormPopup.svelte

```diff
@@ -48,6 +53,22 @@
         isTagInputOpen = true
     }
 
+    let tagError: string = ''
+    function validateTag(): Promise<void> {
+        if (tag.length > 64) {
```

Comment:
TIP-22 specifies the protocol size in byte length, whereas `length` returns the amount of UTF-16 (can be 2 bytes) code units. To get the bytes use something like `Buffer.from(str).size` or `new Blob([str]).size`.

A tag like `€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€` is accepted, but is actually represented by more than 64 bytes.

Same logic applies to metadata

Ref:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length
- https://www.geeksforgeeks.org/how-to-get-the-length-of-a-string-in-bytes-in-javascript/